src/frontend/src/user/user_manager.rs

```diff
@@ -16,45 +16,59 @@ use std::collections::HashMap;
 
 use risingwave_pb::user::{GrantPrivilege, UserInfo};
 
-use crate::user::{UserInfoVersion, UserName};
+use crate::user::{UserId, UserInfoVersion};
 
 /// `UserInfoManager` is responsible for managing users.
 pub struct UserInfoManager {
     version: UserInfoVersion,
-    users: HashMap<UserName, UserInfo>,
+    user_by_name: HashMap<String, UserInfo>,
+    user_name_by_id: HashMap<UserId, String>,
```

Comment:
Is it more natural to map name to id, and then id to info?

Reply:
Well, I followed the mapping rule as same as all `Catalog`'s definition. We can fix all this together in a new PR.